.github/workflows/reusable-release.yaml

```diff
@@ -99,10 +104,11 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.ORG_REPO_TOKEN }}
           NFPM_DEFAULT_RPM_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
           GPG_FILE: "gpg.key"
+          TMPDIR: "tmp"
 
       - name: "remove gpg key"
         run: |
-          rm gpg.key
+          rm gpg.key      
```

Comment:
nit: added spaces?

Reply:
i didn't add them 😄 
removed in https://github.com/aquasecurity/trivy/pull/6533/commits/f59d32ad5083fdc3e53ef63e8935105573d68b01